uint64 timestamp	# Microseconds since system boot
int32[100] string
uint64 MAX_STRLEN = 100
uint64 strlen
